kore/src/Kore/Repl/Data.hs

```diff
@@ -124,6 +125,9 @@ newtype ReplScript = ReplScript
 data ReplMode = Interactive | RunScript
     deriving (Eq, Show)
 
+newtype ScriptModeOutput = ScriptModeOutput Bool
```

Comment:
Any reason this is a newtype over `Bool` instead of a standalone type like the following?
```suggestion
data ScriptModeOutput = EnableOutput | DisableOutput
```